nx-X11/programs/Xserver/hw/nxagent/Dialog.h

```diff
@@ -212,5 +218,29 @@ Ctrl+Alt+E to enable it again.\
 
 #define DIALOG_DISABLE_DEFER_MODE_LOCAL 0
 
+
+#define DIALOG_ENABLE_AUTOGRAB_MODE_MESSAGE \
+\
+"\
+Keyboard auto-grabbing mode is now enabled. You can press\n\
```

Comment:
I don't like that hardcoded keystroke here. Maybe change the text to "press the same keystroke again to..."